src/native/corehost/hostmisc/pal.windows.cpp

```diff
@@ -322,27 +322,18 @@ namespace
     }
 }
 
-bool pal::get_dotnet_self_registered_config_location(pal::string_t* recv)
+void pal::get_dotnet_self_registered_config_location(pal::string_t* recv)
 {
-#if !defined(TARGET_AMD64) && !defined(TARGET_X86)
-    return false;
-#else
```

Comment:
Given that you made basically no changes to existing tests - does it mean that we don't run the tests on Windows ARM32 or ARM64 at all?

Or does it mean that we didn't have any tests validating the current behavior?

Reply:
Apparently the host tests don't run on arm or arm64 at all (Windows or not):
https://github.com/dotnet/runtime/blob/fc4a42746d9cf65ad3db4074207069ab9c168bc9/eng/pipelines/installer/jobs/base-job.yml#L63-L69

Reply:
I'll enable them on a separate PR -- curious if this will cause existing tests to fail.

Reply:
Fair warning - I expect enabling them may be a bit involved. I think the main reason the arm/arm64 tests are currently skipped is that the hosting tests haven't been set up to send tests to helix, so they just run on the build machine (which works fine for x64/x86).